.goreleaser.yml

```diff
@@ -0,0 +1,92 @@
+before:
+  hooks:
+    - go mod tidy
+    - go get github.com/woodpecker-ci/togo
+#    - (cd web/; go generate ./...)
+builds:
+  - id: server
+    main: ./cmd/drone-server
+    binary: woodpecker-server
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      # - windows
+      # - darwin
+  - id: agent
+    main: ./cmd/drone-agent
+    binary: woodpecker-agent
+    env:
+      - CGO_ENABLED=0
```

Comment:
```suggestion
      - CGO_ENABLED=1
```

Reply:
@6543 With `CGO_ENABLED=1` I only get the following error: `standard_init_linux.go:228: exec user process caused: no such file or directory`. In the current makefile CGO is enabled for the server, but not for the agent 🤔

Reply:
well server need the cgo since we have sqlite3 as option to store things

Reply:
changed server release to use cgo with ee8fe2f74b1c6afcff8236b000ed93f80c4d695d